src/murmur/CMakeLists.txt

```diff
@@ -50,12 +50,12 @@ set(MURMUR_SOURCES
 )
 
 if(WIN32)
-	add_executable(murmur WIN32 ${MURMUR_SOURCES})
+	add_executable(mumble-server WIN32 ${MURMUR_SOURCES})
 else()
-	add_executable(murmur ${MURMUR_SOURCES})
+	add_executable(mumble-server ${MURMUR_SOURCES})
```

Comment:
So notably this also changes the executable name for builds on linux and macOS, right?

(Linux distributions already distributed under that name anyway)

Reply:
Yes this changes the server executable's name on all platforms